wiki/Twitter/id.md

```diff
@@ -1,18 +1,24 @@
----
-outdated: true
-outdated_since: a6c6c6b27315fdf9e68f709dab2a08b085428e3b
----
-
 # Twitter
 
-## Kami juga menge-tweet!
+osu! memiliki beberapa akun Twitter, masing-masing melayani tujuan tertentu. Sebagian besar memiliki lalu lintas sangat rendah tetapi akan membantu kamu dalam mendapatkan informasi di luar lingkungan osu!.
+
+## Layanan
+
+| Avatar | Handle | Deskripsi |
+| :-: | :-: | :-- |
+| ![Avatar osu!status](img/osustatus.jpg) | [@osustatus](https://twitter.com/osustatus "Twitter") | Diperuntukkan untuk menyediakan pemberitahuan, dengan lalu lintas rendah, tentang masalah situs dan [Bancho](/wiki/Bancho_(server)). |
+| ![Avatar osu! support](img/osusupport.jpg) | [@osusupport](https://twitter.com/osusupport "Twitter") | Membantu kamu terkait masalah akun/komunitas. Dikelola oleh [account support team](/wiki/People/The_Team/Account_support_team). |
+
+## Komunitas
+
+| Avatar | Handle | Deskripsi |
+| :-: | :-: | :-- |
+| ![Avatar osu!](img/osugame.jpg) | [@osugame](https://twitter.com/osugame "Twitter") | Sumber resmi berita dan pengumuman seputar osu!. |
+| ![Avatar BanchoBoat](img/banchoboat.jpg) | [@banchoboat](https://twitter.com/banchoboat "Twitter") | Bantuan komedi ketika ada yang salah. |
+| ![Avatar NAT](img/osu_nat.png) | [@osu_nat](https://twitter.com/osu_nat "Twitter") | Berita, pengumuman, dan survey kecil-kecilan yang dikelola oleh [NAT](/wiki/People/The_Team/Nomination_Assessment_Team) (tidak dikelola oleh [osu!team](/wiki/People/The_Team#the-team)). |
 
-osu! memiliki beberapa akun Twitter, masing-masing melayani tujuan tertentu. Sebagian besar lalu lintas sangat rendah tetapi akan membantu anda tetap mendapat informasi di luar lingkungan osu!.
+## Pribadi
 
-| Avatar | Handle | Dekripsi |
+| Avatar | Handle | Deskripsi |
 | :-: | :-: | :-- |
-| ![Avatar osu!](img/osugame.jpg) | [@osugame](https://twitter.com/osugame) | Sumber berita resmi dan pengumuman. Trafik rendah namun berkualitas. |
-| ![Avatar osu!status](img/osustatus.jpg) | [@osustatus](https://twitter.com/osustatus) | Diperuntukkan untuk menyediakan pemberitahuan lalu lintas rendah tentang masalah situs dan bancho. |
-| ![Avatar Dean Herbert](img/ppy.jpg) | [@ppy](https://twitter.com/ppy) | Tidak sebatas osu! saja, tapi kami akan meninggalkan ini di sini bagi mereka yang tertarik. Lagipula, kebanyakan kehidupan ppy adalah osu!. |
-| ![Avatar osu! support](img/osusupport.jpg) | [@osusupport](https://twitter.com/osusupport) | Membantu anda dengan masalah akun/komunitas. |
-| ![Avatar BanchoBoat](img/banchoboat.jpg) | [@banchoboat](https://twitter.com/banchoboat) | Bantuan komedi ketika ada yang salah. |
+| ![Avatar Dean Herbert](img/ppy.jpg) | [@ppy](https://twitter.com/ppy "Twitter") | Akun pribadi [pembuat osu!](/wiki/People/peppy), yang berisi konten yang tidak selalu terkait osu!, namun masih membahas banyak hal terkait osu! juga. |
```

Comment:
```suggestion
| ![Avatar Dean Herbert](img/ppy.jpg) | [@ppy](https://twitter.com/ppy "Twitter") | Akun Twitter pribadi [pembuat osu!](/wiki/People/peppy), yang mana berisi konten tidak selalu terkait osu!, namun masih membahas banyak hal terkait osu! juga. |
```

Reply:
added detail information which account ppy have and remove double word in a sentence.